src/Sylius/Bundle/CoreBundle/spec/Form/Type/ProductReviewAdminTypeSpec.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace spec\Sylius\Bundle\CoreBundle\Form\Type;
+
+use PhpSpec\ObjectBehavior;
+use Prophecy\Argument;
+use Symfony\Component\Form\FormBuilderInterface;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class ProductReviewAdminTypeSpec extends ObjectBehavior
+{
+    function let()
+    {
+        $this->beConstructedWith('dataClass', array('validation_group'), 'product');
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\CoreBundle\Form\Type\ProductReviewAdminType');
+    }
+
+    function it_extends_product_review_admin_type()
+    {
+        $this->shouldHaveType('Sylius\Bundle\CoreBundle\Form\Type\ProductReviewType');
+    }
+
+    function it_builds_form(FormBuilderInterface $builder)
+    {
+        $builder
+            ->add('rating', 'choice', array(
```

Comment:
We have agreed that last parameter in form builder spec should be mocked with prophecy -> `Argument::cetera()`